ServerCore/Pages/Resources/nipd2023/FAQPartial.cshtml

```diff
@@ -0,0 +1,164 @@
+        </p>
+
+        <h3>
+            How do we solve puzzles that don't have instructions?
+        </h3>
+        <p>
+            The implicit instruction for all puzzles is &quot;Figure out the final answer.&quot; Puzzles without instructions are either classic puzzle forms, perhaps with a twist, or puzzles where the challenge is figuring out how to solve the puzzle. In the latter case, the key is usually to look for patterns or common bonds between the parts of the puzzle. For example, are many of the words spelled wrong? Have all the vowels been left out? Sometimes there may be a hint hidden in the puzzle.  You may also find inspiration by looking at the list of <a asp-page="/EventSpecific/Encodings">Encodings</a> or <a asp-page="/EventSpecific/Tools">Tools</a>.
+        </p>
+
+        <h3>
+            What's the best way to win?
+        </h3>
+        <p>
+            The best way to win is to solve puzzles quickly so that you get to the final metapuzzle. The best way to accomplish this is with teamwork. Many teams prefer to work individually on the easier puzzles and work as a group on the harder puzzles. If you get stuck, ask for help or switch to another puzzle. Make sure you keep track of which puzzles you've already solved and confirmed using the answer submission system.
+        </p>
+
+        <h3>
+            What's the best way to lose?
+        </h3>
+        <p>
+            Not having fun. If you're not having fun working on a particular puzzle, switch to another one.
+        </p>
+
+        <h3>
+            What are the prizes?
+        </h3>
+        <p>
+            Solving puzzles is its own reward!
+        </p>
+
+        <h3>
+            Can I finish by accident, without solving the puzzles?
+        </h3>
+        <p>
+            Haha. No.
+        </p>
+
+        <h3>
+            Do I have to register to participate?
+        </h3>
+        <p>
+            Yes. If you do miss the registration deadline, send mail to NIPD staff at <a href="mailto:puzzhunt@microsoft.com">puzzhunt</a>.
+        </p>
+
+        <h3>
+            Do I have to be on the Redmond campus to participate?
+        </h3>
+        <p>
+            No! This event is fully completable and fully enjoyable remotely.  There are a few puzzles that you have to be physically present for on campus to interact with people or items only available on campus, but the event is still fully finishable even if those puzzles are skipped.
+        </p>
+
+        <h3>
+            If I'm on campus, where am I allowed to go?
+        </h3>
+        <p>
+            Since all of the in-person components take place in Building 99, you'll probably want to stay west of 520.  Additionally, Studios B, C, X, Buildings 92, 112, 114, 120, 123 - 127, 25, 34, 35, 41, 42, 87, and all construction areas are OFF LIMITS and should not be entered under any circumstances.
```

Comment:
Is this the current list?

Reply:
Yes, it's the one they gave us this year, and we're assuming it won't have changed in the past 2 weeks.